roles/pulp_devel/templates/alias.bashrc.j2

```diff
@@ -225,7 +225,9 @@ pfixtures () {
     cat ~/.config/pulp_smash/settings.json | \
     jq "setpath([\"custom\",\"fixtures_origin\"]; \"http://localhost:8080/fixtures/\")" > ~/.config/pulp_smash/.settings.json.tmp
     mv ~/.config/pulp_smash/.settings.json.tmp ~/.config/pulp_smash/settings.json
+    export REMOTE_FIXTURES_ORIGIN="http://localhost:8080/fixtures/"
     podman run --rm -it -p 0.0.0.0:8080:8080 --name pulp_fixtures pulp/pulp-fixtures
+    export REMOTE_FIXTURES_ORIGIN={{ pulp_remote_fixtures_origin }}
```

Comment:
This might not be needed since we cannot change environment variables from a child process.

Also, I think there is no need to create any workarounds to change the environment variable since we are not planning to support the installer as a main development environment. Am I right, @mikedep333? The other changes would be fine to be merged.

Reply:
You are correct that the installer is no longer the main development environment. In fact, it is going away soon entirely.